src/Symfony/Component/Console/Resources/completion.bash

```diff
@@ -0,0 +1,33 @@
+# This file is part of the Symfony package.
+#
+# (c) Fabien Potencier <fabien@symfony.com>
+#
+# For the full copyright and license information, please view
+# https://symfony.com/doc/current/contributing/code/license.html
+
+_sf_{{ COMMAND_NAME }}() {
+    local sf_cmd="${COMP_WORDS[0]}"
+    if [ ! -f "$sf_cmd" ]; then
+        return 1
+    fi
+
+    local cur prev words cword
+    _get_comp_words_by_ref -n := cur prev words cword
+
+    local completecmd=("$sf_cmd" "_complete" "-sbash" "-c$cword" "-v{{ VERSION }}")
+    for w in ${words[@]}; do
+        completecmd+=(-i "'$w'")
+    done
+
+    local sfcomplete=$(${completecmd[@]})
+    if [ $? -ne 0 ]; then
+        echo $sfcomplete
```

Comment:
Noted this before: if you use this feature on a Symfony app where complete is not available (older Symfony), it should just not work. The error shouldn't be presented as completion options.

Reply:
I would love to, but I don't know how. There is no special exit code and the errors are logged to stderr.

Reply:
What happens if you try to complete on a copy of Symfony which doesn't have completion? Say, <= 5.3? My comment is just this should fail somewhat gracefully, it may already work.

Reply:
Yeah, it's not graceful at the moment :cry: 

![image](https://user-images.githubusercontent.com/749025/132834967-cea2ee64-035a-49e4-80dd-fa4e13262b39.png)

Reply:
Can we maybe capture the STDERR and check it?